pyvista/utilities/fileio.py

```diff
@@ -187,7 +199,13 @@ def read(filename, attrs=None):
             reader = get_reader(filename)
             return standard_reader_routine(reader, filename)
         except KeyError:
-            pass
+            # Attempt read with meshio
+            try:
+                from meshio._helpers import _extension_to_filetype
+                return read_meshio(filename, _extension_to_filetype[ext])
+            except:
+                pass
```

Comment:
Perhaps we should throw an error or warning message  here if `meshio` is not installed - if it is not installed, then a user will just see our uninformative error at the end of this method

Reply:
Will ``meshio`` be a dependency of ``pyvista``? If so, it should be installed when installing ``pyvista``, right?

Reply:
Ah yes - can you go ahead and add `meshio` to the `setup.py` and in the `requirements.txt`? Do you know if we should specify above a specific version?

Reply:
And can you also list it in the "getting started" page in the docs?